libs/stx/inc/public/tfc/utils/units_glaze_meta.hpp

```diff
@@ -178,6 +178,8 @@ inline consteval auto dimension_name() -> std::string_view {
     return "pressure";
   } else if constexpr (mp_units::convertible(ref_t, mp_units::si::kelvin)) {
     return "temperature";
+  } else if constexpr (mp_units::convertible(ref_t, mp_units::si::litre / mp_units::si::second)) {
+    return "flow";
```

Comment:
Could you call this `volumetric_flow`  and remove the comment above with volumetric_flow L#120.

Reply:
finished